main.nf

```diff
@@ -30,8 +31,8 @@ params.fasta_index = params.genome ? params.genomes[ params.genome ].fasta_index
 assert params.aligner == 'bwameth' || params.aligner == 'bismark' : "Invalid aligner option: ${params.aligner}. Valid options: 'bismark', 'bwameth'"
 
 Channel
-    .fromPath("$baseDir/assets/where_are_my_files.txt", checkIfExists: true)
-    .into { ch_wherearemyfiles_for_trimgalore; ch_wherearemyfiles_for_alignment }
+    .fromPath("$baseDir/assets/where_are_my_files.txt", checkIfExists: false)
+    .into {ch_wherearemyfiles_for_fastqc; ch_wherearemyfiles_for_trimgalore; ch_wherearemyfiles_for_alignment }
```

Comment:
I'm not sure why you need a `ch_wherearemyfiles_for_fastqc` for FastQC? These are placeholders for steps where we don't save the output files to `.results` by default. We always save FastQC reports, so no need for it there.